plugins/woocommerce-admin/client/layout/controller.js

```diff
@@ -215,6 +221,22 @@ export const getPages = () => {
 		} );
 	}
 
+	if ( window.wcAdminFeatures[ 'wc-pay-subscriptions-page' ] ) {
```

Comment:
Is the plan to take over this page in WC Pay once its activated?  I get that this page needs to be registered here since WC Pay is not yet installed at this point, but hoping post-install code can be extended within the WC Pay plugin.

Reply:
>Is the plan to take over this page in WC Pay once its activated?

That's correct. WooCommerce Payments registers its own "Subscriptions" admin menu/page via its bundled subscriptions functionality (i.e. https://github.com/automattic/woocommerce-subscriptions-core).

The Subscriptions page looks like this when WCPay is active:

<img width="1157" alt="Screen Shot 2022-05-18 at 9 57 00 am" src="https://user-images.githubusercontent.com/1527164/168930880-5e4c05b3-34f2-4ca5-8380-7321c4c862e6.png">

>but hoping post-install code can be extended within the WC Pay plugin.

Could you clarify this point? I'm not sure what you mean.

Reply:
>Could you clarify this point? I'm not sure what you mean.

Meaning that the actual Subscriptions page is housed in WC Pay after install and overrides the one registered here, which looks to be the case from your above comment. 😄